src/sentry/integrations/vsts/webhooks.py

```diff
@@ -62,6 +70,29 @@ def handle_assign_to(self, integration, external_issue_key, assigned_to):
             assign=assign,
         )
 
+    def handle_status_change(self, integration, external_issue_key,
+                             status_change, project):
+        if status_change is None:
+            return
+        new_state = status_change.get('newValue')
+        old_state = status_change.get('oldValue')
+        organization_ids = OrganizationIntegration.objects.filter(
+            integration_id=integration.id,
+        ).values_list('organization_id', flat=True)
+
+        old_category = None
+        new_category = None
+        for organization_id in organization_ids:
+            installation = integration.get_installation(organization_id)
+            if not (old_category and new_category):  # states should be the same throughout
+                old_category, new_category = installation.get_state_categories(
```

Comment:
should we really be caching these across organizations? since in theory, the `default_auth_id` could be different for different orgs, it seems safer to not cache them. what if one org's default auth is no longer valid? i don't think we should use another org's auth to pull that data

Reply:
Gotcha!